xsql/parser_test.go

```diff
@@ -26,7 +26,18 @@ func TestParser_ParseStatement(t *testing.T) {
 				Sources: []Source{&Table{Name: "tbl"}},
 			},
 		},
-
+		{
+			s: "SELECT `select` FROM tbl",
```

Comment:
1. Add parse error unit case - such as 'SELECT `select FROM tbl'
2. Add Chinese, and field name with blank character etc.